Browsable FAQs/browsable_faqs.ex

```diff
@@ -0,0 +1,96 @@
+stack ContentRepo, "Browsable FAQs with Content Repo integraion" do
+    card MainMenu, "Main Menu" do
+        # The main menu is a special case, since it is not a specific content page, but rather a
+        # collection of content indexes, that we can select using a specific tag
+        # CONFIG: You can modify the content repo URL here to point to a different content repo instance
+        contentrepo_url = "https://content-repo-api.qa.momconnect.co.za"
+        # CONFIG: You can modify the main menu tag here, to select for a different tag that represents your main menu
+        main_menu_tag = "mainmenu"
```

Comment:
would we not want the trigger here instead of the tag?

Reply:
So I wasn't sure about trigger vs tag. This is the way YAL does it, and MomConnect uses a different suggested topics API, so I went with this one. I haven't seen the trigger in use, how does that work vs tag?

Reply:
I see the `trigger` query param is not supported in the version of contentrepo that MomConnect is using, so I'm going to leave it like this for now.